CHANGES

```diff
@@ -9,6 +9,9 @@
 
  Changes between 1.1.0f and 1.1.1 [xx XXX xxxx]
 
+  *) Support multi-prime RSA.
```

Comment:
Side note: you could mention the RFC here.  Your choice

Reply:
seems reasonable